langchain4j-parent/pom.xml

```diff
@@ -127,6 +127,12 @@
                 <artifactId>snakeyaml</artifactId>
                 <version>2.0</version>
             </dependency>
+
+            <dependency>
+                <groupId>com.alibaba</groupId>
+                <artifactId>dashscope-sdk-java</artifactId>
```

Comment:
Could you please move all Qwen-related code to a separate `langchain4j-qwen` module? Same like `langchain4j-pinecone`, `langchain4j-weaviate`, etc.

Reply:
OK, I will move the implementation to a separate module.
About the module name, "qwen" is a name of model family (just like "gpt-3.5/4..." in openai), and dashscope is a model service provider (just like openai?). Furthermore, modelscope is a model community (just like huggingface).
Should I use langchain4j-dashscope here?